warehouse/table_upload.go

```diff
@@ -57,45 +59,31 @@ func areTableUploadsCreated(uploadID int64) bool {
 }
 
 func createTableUploads(uploadID int64, tableNames []string) (err error) {
-
-	//Using transactions for bulk copying
-	txn, err := dbHandle.Begin()
-	if err != nil {
-		return
-	}
-
-	stmt, err := txn.Prepare(pq.CopyIn(warehouseutils.WarehouseTableUploadsTable, "wh_upload_id", "table_name", "status", "error", "created_at", "updated_at"))
-	if err != nil {
-		pkgLogger.Errorf(`[WH]: Error preparing txn: %v Error: %v`, stmt, err)
-		return
-	}
-	defer stmt.Close()
-
-	now := timeutil.Now()
-	for _, table := range tableNames {
-		_, err = stmt.Exec(uploadID, table, "waiting", "{}", now, now)
-		if err != nil {
-			pkgLogger.Errorf(`[WH]: Error copying row in pq.CopyIn for table: %s Error: %v`, table, err)
-			txn.Rollback()
-			return
+	columnsInInsert := 6
```

Comment:
Instead of hardcoding the number, let's do something like this.

```
tableUploadColumns := ["uploadID", "state", "tableName", ..]
columnsInInsert := len(tableUploadColumns)
```

Reply:
changed